### PR TITLE
Add MR test

### PR DESCRIPTION
**Pull request by nebraszka**

### CODE REVIEW

File: test/src/graph/multiReturnTest.cpp [hunk 1]
```diff
@@ -1,40 +1,330 @@
...
+
+	const rgl_mat3x4f lidarPose{(Mat3x4f::translation(lidarTransl + stairsTranslation)).toRGL()};
+
+	// Lidar with MR
+	const float beamDivAngle = vlp16LidarHBeamDivergence;
+	constructMRGraph(raysTf, lidarPose, beamDivAngle, true);
+
+	// Camera
+	rgl_mat3x4f cameraPose = Mat3x4f::translation({0.0f, -1.5f, stepHeight * 3 + 1.0f}).toRGL();
+	constructCameraGraph(cameraPose);
```
Comment (PawelLiberadzki) [inline test/src/graph/multiReturnTest.cpp:284]: Camera graph does not seem to be run?

File: test/src/graph/multiReturnTest.cpp [hunk 2]
```diff
@@ -1,40 +1,330 @@
...
+#endif
+		EXPECT_RGL_SUCCESS(rgl_graph_node_add_child(mrRays, mrTransform));
+		EXPECT_RGL_SUCCESS(rgl_graph_node_add_child(mrTransform, mrRaytrace));
+		EXPECT_RGL_SUCCESS(rgl_graph_node_add_child(mrRaytrace, mrFirst));
+		EXPECT_RGL_SUCCESS(rgl_graph_node_add_child(mrRaytrace, mrLast));
+		EXPECT_RGL_SUCCESS(rgl_graph_node_add_child(mrFirst, compactFirst));
+		EXPECT_RGL_SUCCESS(rgl_graph_node_add_child(mrLast, compactLast));
+		EXPECT_RGL_SUCCESS(rgl_graph_node_add_child(compactFirst, mrFormatFirst));
+		EXPECT_RGL_SUCCESS(rgl_graph_node_add_child(compactLast, mrFormatLast));
+		if (withPublish) {
```
[the remark this answers is not in the document]
Comment (nebraszka) [reply test/src/graph/multiReturnTest.cpp:79]: If a compilation without ROS2 with `withPublish==true` occurs, then `GTEST_SKIP` will be called, preventing further test execution at runtime, `rgl_graph_node_add_child` will not be called.

File: test/src/graph/multiReturnTest.cpp (hunk 1 above)
Comment (nebraszka) [reply test/src/graph/multiReturnTest.cpp:284]: [Here](https://github.com/RobotecAI/RobotecGPULidar/pull/290/files#diff-536edcfe4b5e3529fc9e502b5fbb93a51c7f50cf1f2a5d623ec6f336c62720e1R291)

Comment (PawelLiberadzki) [reply test/src/graph/multiReturnTest.cpp:287]: Ok, that's clear, thank you for explanation!

**Review by PawelLiberadzki — APPROVED** [review]
LGTM! But please merge after https://github.com/RobotecAI/RobotecGPULidar/pull/295, because these tests are already updated to the change from that PR.

**Pull request merged**